lib/minimize.js

```diff
@@ -34,7 +34,7 @@ function Minimize(parser, options) {
   // Prepare the parser.
   //
   this.htmlparser = parser || new html.Parser(
-    new html.DomHandler(this.emits('read'))
+    new html.DomHandler(this.emits('read')), this.helpers.htmlParserOptions()
```

Comment:
could you just have this be `this.options.dom`, where `this.options.dom = options.dom || {};` is set in the constructor function. That allow for more domparser options and removes the need for the `htmlParserOptions` method above. Would be nice to separate that option from the default options for minimize itself.